quest/events.py

```diff
@@ -39,6 +39,8 @@ def __setitem__(self, key: str, value: Event): ...
 
     def __contains__(self, key: str) -> bool: ...
 
+    def __len__(self) -> int: ...
```

Comment:
Because this function is only used in testing, I would leave it off the interface. Just have the tests reach in do `len(..._events._state)`